src/synth/fluid_synth.c

```diff
@@ -3652,15 +3652,24 @@ fx[ ((k * fluid_synth_count_effects_channels() + j) * 2 + 1) % nfx ]  = right_bu
  *
  * @param synth FluidSynth instance
  * @param len Count of audio frames to synthesize and store in every single buffer provided by \p out and \p fx.
- * @param nfx Count of arrays in \c fx. Must be a multiple of 2 (because of stereo)
- * and in the range <code>0 <= nfx/2 <= (fluid_synth_count_effects_channels() * fluid_synth_count_effects_groups())</code>.
+ *  Zero value is permitted, the function does nothing and return FLUID_OK.
+ * @param nfx Count of arrays in \c fx. Must be a multiple of 2 (because of stereo).
+ * and in the range <code>0 <= nfx/2 <= (fluid_synth_count_effects_channels() * fluid_synth_count_effects_groups()).
+  Note that zero value is valid and allows to skip mixing effects in all fx output buffers.</code>.
  * @param fx Array of buffers to store effects audio to. Buffers may
-alias with buffers of \c out. NULL buffers are permitted and will cause to skip mixing any audio into that buffer.
+alias with buffers of \c out. Individual NULL buffers are permitted and will cause to skip mixing any audio into that buffer.
  * @param nout Count of arrays in \c out. Must be a multiple of 2
-(because of stereo) and in the range <code>0 <= nout/2 <= fluid_synth_count_audio_channels()</code>.
+(because of stereo) and in the range <code>0 <= nout/2 <= fluid_synth_count_audio_channels().
+ Note that zero value is valid and allows to skip mixing dry audio in all out output buffers.</code>.
```

Comment:
```suggestion
(because of stereo) and in the range <code>0 <= nout/2 <= fluid_synth_count_audio_channels()</code>.
 Note that zero value is valid and allows to skip mixing dry audio in all out output buffers.
```